mlflow/lightgbm.py

```diff
@@ -158,8 +158,7 @@ def load_model(model_uri):
                       `Referencing Artifacts <https://www.mlflow.org/docs/latest/tracking.html#
                       artifact-locations>`_.
 
-    :return: An `lightgbm.Booster model object
```

Comment:
Ah, I think the old doc here was actually fine, sorry for the confusion - seems nice to link to the lightgbm docs, but this is also fine as is.